File: server/src/main/java/com/example/server/dto/offer/OfferResponseDto.java
package com.example.server.dto.offer;

import lombok.AllArgsConstructor;
import lombok.Builder;
import lombok.Data;
import lombok.NoArgsConstructor;

import java.time.LocalDateTime;
import java.util.List;

@Data
@Builder
@NoArgsConstructor
@AllArgsConstructor
public class OfferResponseDto
{

  private String id;
  private String title;
  private String description;
  private int originalPrice;
  private int offerPrice;
  private int numberOfItems;
  private int numberOfAvailableItems;
  private LocalDateTime pickupTimeStart;
  private LocalDateTime pickupTimeEnd;
  private List<String> categories;
  private String imagePath;
  private String status;
  private String businessId;
  private String businessName;
  private String businessLogoPath;

}

File: client/client/Dto/Offer/OfferResponseDto.cs
namespace client.Dto.Offer;

public class OfferResponseDto
{
    public string Id { get; set; }
    public string Title { get; set; }
    public string Description { get; set; }
    public int OriginalPrice { get; set; }
    public int OfferPrice { get; set; }
    public int NumberOfItems { get; set; }
    public int NumberOfAvailableItems { get; set; }
    public DateTime PickupTimeStart { get; set; }
    public DateTime PickupTimeEnd { get; set; }
    public List<string> Categories { get; set; }
    public string ImagePath { get; set; }
    public string Status { get; set; }
    public string BusinessId { get; set; }
    public string BusinessName { get; set; }
    public string BusinessLogoPath { get; set; }
    public AddressDto BusinessAddress { get; set; }
}